apps/ledger-live-desktop/src/renderer/screens/settings/sections/Experimental/StorylyTester.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useCallback } from "react";
+import { useTranslation, Trans } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { SettingsSectionRow } from "~/renderer/screens/settings/SettingsSection";
+import Button from "~/renderer/components/Button";
+import { openModal } from "~/renderer/actions/modals";
+
+const LottieTester = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const onOpenModal = useCallback(() => dispatch(openModal("MODAL_STORYLY_DEBUGGER")), [dispatch]);
+
+  return (
+    <SettingsSectionRow title={t("settings.experimental.features.testStories.title")} desc="">
+      <Button onClick={onOpenModal} primary>
+        <Trans i18nKey={"storylyDebugger.buttonTitle"} />
```

Comment:
no change needed here but just note that usually the `t` function does the job and you don't actually need `<Trans />`
https://react.i18next.com/latest/trans-component#important-note